test/e2e/storage/persistent_volumes-local.go

```diff
@@ -258,6 +259,9 @@ var _ = SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [S
 		AfterEach(func() {
 			cleanupLocalVolume(config, testVol)
 		})
+		if selinux.SELinuxEnabled() {
+			framework.Skipf("SELinux is enabled, this test will fail, skeeping it")
```

Comment:
if e2e tests are running on a platform where SELinux enabled then this test will fail as two pods will automatically have assigned different selinux lables and as they try to access the same volume at the same time, it will fail. On host with SELinux disabled, this test will work fine.